app/src/main/java/com/codesoom/assignment/utils/JwtUtil.java

```diff
@@ -18,13 +21,25 @@ public JwtUtil(@Value("${jwt.secret}") String secret) {
         key = Keys.hmacShaKeyFor(secret.getBytes());
     }
 
+    /**
+     * 주어진 유저 식별자로 부호화한 JWT Token을 반환한다.
```

Comment:
이건 머리를 비우고 읽으면 토큰 객체를 리턴하는 걸로 생각할 수도 있을 것 같아요.
```suggestion
     * 주어진 유저 식별자로 부호화한 JWT Token 문자열을 반환한다.
```

Reply:
미처 생각지 못했네요 .. 오해의 소지가 분명 있을 것 같습니다.